application/src/main/resources/thingsboard.yml

```diff
@@ -639,6 +639,36 @@ redis:
   db: "${REDIS_DB:0}"
   # db password
   password: "${REDIS_PASSWORD:}"
+  # ssl config
+  ssl:
+    # Enable/disable secure connection
+    enabled: "${TB_REDIS_SSL_ENABLED:false}"
+    # Server SSL credentials
+    credentials:
+      # Server credentials type (PEM - pem certificate file; KEYSTORE - java keystore)
```

Comment:
Either change description and use pem and keystore in lowercase, or use `equalsIgnoreCase` in `createAndInitTrustManagerFactory`.